tests/plugins/test_check_and_set_platforms.py

```diff
@@ -160,10 +161,14 @@ def test_check_and_set_platforms(tmpdir, platforms, platform_exclude, platform_o
 
     plugin_result = runner.run()
     if platforms:
+        assert "Koji platforms are" in caplog.text()
+        for platform in platforms:
+            assert platform in caplog.text()
```

Comment:
If I understand correctly, this seems like a pretty weak check, since the platform string could appear anywhere in the log - not necessarily associated with "Koji platforms are" - if  plugin args are logged or something - maybe have a helper function that you use for all the added test: 

    def assert_platforms_message(message, caplog, expected_platforms):
         matched = re.match(message + r' (\[[^\]]*\])', caplog.text())
         if not matched:
            raise RuntimeError("'{} [....]' not found in log".format(message))
         if str(expected_platforms) != matched.group(1):
             raise RuntimeError("Expected {} {}, found {}".format(message, expected_platforms, matched.group(1))

(Untested - something like that)

Reply:
Or I could just sorted() the platforms in the log messages and add the sorted platforms to the message I'm looking for in the assert.